app/posts/find-teacher-training/2021-08-19-new-subject-flow.md

```diff
@@ -0,0 +1,83 @@
+
+## New flow
+
+To resolve the issues, we have added an initial question about age group. This allows us to then present a different list of subjects depending upon which age group was selected.
+
+![Flow diagram showing how the new age group and subject question pages are linked together](subject-flow.png)
+
+If a user selects ‘Primary’, we ask a follow-up question about whether they would like to see specialist subjects or not. If they answer Yes, we show the primary specialist subjects, as well as the option to see primary courses without a specialist subject.
+
+If a user selects ‘Secondary’, we show all the secondary subjects a single alphabetical list. This proved to be simpler and quicker than separating out the language subjects.
+
+If a user selects ‘Further education’, then no further course subject questions are required.
+
+![Screenshot of new page askind users what age group they are interested in teaching](age-group.png)
+
+## Special educational needs and disability (SEND)
+
+Previously, the ‘Show only courses with a SEND specialism’ checkbox worked as [an additional filter](/find-teacher-training/sen-filter/) to the other subject checkboxes. This meant that if you ticked it as well as a subject, only courses had that subject AND a SEND-specialism were shown. This makes it behave differently from the other subject checkboxes.
+
+We initially explored whether we could treat SEND as a subject, and have it behave the same way as other subjects. This makes some sense for primary courses which have specialisms, but less sense for secondary courses.
+
+Instead, the simplest initial thing to do is to remove SEND as a checkbox from both the subject pages altogether, and keep it as a separate filter on the results page.
+
+In future, we can review usage of this filter, alongside other issues around SEND-specialist courses (such as whether it is used consistently by providers) to decide whether or not to make further changes.
+
+## Primary courses
+
+If someone wants to teach the primary age group, we ask them if want to find courses with a specialist subject. Having this as a separate question lets us explain what this means, and also means that users who answer ‘No’ can skip straight to the list of courses.
+
+![Screenshot of the new page asking users if they would like to find Primary courses that include a specialist subject](primary.png)
+
+If a user answers yes, they are shown a list of the specialist subjects. A checkbox at the bottom allows them to also see primary courses without a specialist subject.
+
+![Screenshot of the new page asking users what specialist subjects they are interested in](primary-specialist.png)
+
+## Secondary courses
+
+If someone wants to teach the secondary age group, we show them a list of all the subjects in alphabetical order. Whilst this is a long list, we found in usability tests that users were able to find the subjects they were interested in quickly.
+
+![](secondary.png)
```

Comment:
Should we add alt text for this image?

Reply:
Yep, opps - this was to re-test the auto-linter.